src/main/java/com/minecolonies/coremod/placementhandlers/PlacementHandlers.java

```diff
@@ -127,8 +127,13 @@ public Object handle(
                     return ActionProcessingResult.DENY;
                 }
                 placer.handleBuildingOverBlock(pos);
+
+                if (!world.setBlockState(pos, Blocks.DIRT.getDefaultState(), 0x03))
```

Comment:
please document what `0x03` means as a flag by giving it a constant at a prominent reusable place